src/settings/MappingProfiles/MappingProfilesContainer/MappingProfilesContainer.js

```diff
@@ -30,32 +38,28 @@ const customProperties = getMappingProfilesColumnProperties({
   ],
 });
 
+const queryTemplate = '(sortby "%{query.query}")';
+const sortMap = {
+  name: 'name',
+  folioRecord: 'recordTypes',
+  format: 'outputFormat',
+  updated: 'metadata.updatedDate',
+  updatedBy: 'userInfo.firstName userInfo.lastName',
+};
+
 const MappingProfilesContainer = ({
   history,
   match,
   location,
+  mutator,
+  resources,
 }) => {
   return (
     <>
       <MappingProfiles
-        // TODO: temp solution to simulate mutators and resources that should be removed after integration with backend
-        parentResources={{
-          query: {
-            sort: new URLSearchParams(window.location.search).get('sort') || '',
-            query: new URLSearchParams(window.location.search).get('query') || '',
-          },
-          mappingProfiles: {
-            records: tempData.profiles,
-            hasLoaded: true,
-            isPending: false,
-            other: { totalRecords: 4 },
-          },
-        }}
-        parentMutator={{
-          resultCount: { replace: noop },
-          resultOffset: { replace: noop },
-        }}
-        formatter={getMappingProfilesItemFormatter({ format: record => record.format })}
+        parentResources={resources}
+        parentMutator={mutator}
+        formatter={getMappingProfilesItemFormatter({ format: ({ outputFormat }) => (outputFormat || null) })}
```

Comment:
If there won't be anything I think `undefined` would result in empty stuff in the DOM, so we can simplify here.
```suggestion
        formatter={getMappingProfilesItemFormatter({ format: ({ outputFormat }) => outputFormat })}
```